content/articles/cloudflare-service.markdown

```diff
@@ -16,10 +24,45 @@ CloudFlare has two integration methods:
 
+### Direct Integration
+
+Direct integration requires to change the name servers for your domain to point to CloudFlare name servers.
+
+If your domain is registered with DNSimple, follow these instructions to [configure the name servers for a domain](/articles/setting-name-servers/). If the domain is registered elsewhere, you will need to follow the instructions of your existing domain registrar to point the domain to CloudFlare. In both cases, once the domain is pointing to CloudFlare, you will need to manage the DNS records in CloudFlare. Any DNS change in your DNSimple account for that domain will no longer have any effect, including changing DNS records, enabling/disabling one-click services and enabling/disabling email forwards.
+
+Furthermore, the direct integration doesn't allow selectively delegating only certain DNS records to CloudFlare. If you want to use CloudFlare only for a specific DNS record (e.g. for the main website) and use DNSimple for all the other records (e.g. MX email configurations), then use the one-click DNSimple integration.
+
+### DNSimple one-click integration
+
+DNSimple one-click integration allows you to use CloudFlare to proxy only a specific DNS record of a domain. This is the preferred option, for example, if you want to take advantage of CloudFlare's CDN and protection for the main website, maintaining the other DNS records in DNSimple.
+
+However, this approach has some limitations. For instance, CloudFlare doesn't allow to use the Universal SSL feature for domains or subdomains managed from a third-party provider. More information below.
+
+
 ## DNS limitations
 
-CloudFlare caches your site content at locations across the globe. It uses DNS to route your requests to one of these locations depending on various factors. To support this technology within DNS limitations it is best to use a host name (i.e. www.example.com) rather than your domain apex (i.e. example.com). CNAME records at a domain apex are disallowed or will cause your DNS respond incorrectly. You may wish to use DNSimple's URL redirection record to forward requests from your domain apex to "www" for instance.
+CloudFlare caches your site content at locations across the globe. It uses DNS to route your requests to one of these locations depending on various factors. To support this technology within DNS limitations it is best to use a host name (i.e. www.example.com) rather than your domain apex (i.e. example.com). CNAME records at a domain apex are disallowed or will cause your DNS respond incorrectly. You may wish to use DNSimple's URL redirection record to forward requests from your domain apex to "www" for instance.
```

Comment:
I'm a little confused by this sentence:

> To support this technology within DNS limitations it is best to use a host name (i.e. www.example.com) rather than your domain apex (i.e. example.com).

I think we definitely need a comma after `limitations`, but I'm not sure I completely understand what we mean by `DNS limitations`.